.expeditor/publish-release-notes.sh

```diff
@@ -38,6 +44,8 @@ This release uses:
 - Chef Infra Server version
 - Chef InSpec version
 
```

Comment:
Wondering if we can add a section to list down the version of the following components Automate uses:
- Postgres
- ElasticSearch
- Nginx
- Haproxy

This can give the users a view of the underlying softwares we are using and also if these are the ones customers need to know as they directly/indirectly touches while configuring external data stores.

Although, having said that, are we exposing these versions anywhere in any other docs?

Reply:
Are those in the manifest?